mailbox/api/src/main/java/org/apache/james/mailbox/exception/InvalidAnnotationException.java

```diff
@@ -21,7 +21,7 @@
 
 public class InvalidAnnotationException extends Exception {
 
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 3765642311997978478L;
```

Comment:
From @mbaechler : "The better you can do is not define any serialVersionUID at all"
I agree with this :-)